.github/workflows/scripts/check_docstrings.py

```
@@ -0,0 +1,772 @@
+                action="""\
+Adjust your docstring to match the listed function arguments.""",
+            )
+        )
+        return violations
+
+    ######################################################################
+    # Logic below only works when both the function and doctring have args
+    ######################################################################
+
+    # Check whether docstring arguments match function arguments
+    for argument_function in arguments_function:
```

Ruff (0.8.2)

Loop control variable `argument_function` not used within loop body

(B007)